DatadogCore/Tests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -12,67 +12,48 @@ import DatadogInternal
 @testable import DatadogCore
 
 class CrashReportReceiverTests: XCTestCase {
-    private var core: PassthroughCoreMock! // swiftlint:disable:this implicitly_unwrapped_optional
-
-    override func setUp() {
-        super.setUp()
-        core = PassthroughCoreMock()
-    }
-
-    override func tearDown() {
-        core = nil
-        super.tearDown()
-    }
+    private let featureScope = FeatureScopeMock()
 
     func testReceiveCrashEvent() throws {
         // Given
-        let core = PassthroughCoreMock(
-            bypassConsentExpectation: expectation(description: "Send Event Bypass Consent"),
-            messageReceiver: CrashReportReceiver.mockAny()
-        )
+        let receiver: CrashReportReceiver = .mockWith(featureScope: featureScope)
```

Comment:
This is a common pattern in this PR. We change from testing message receivers as part of mocked core to testing them as individual units. 🧽 🧹